package.json

```diff
@@ -20,13 +20,18 @@
     "lint:js:fix": "eslint . --fix",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
-    "test:ts": "tsc --noEmit",
     "start": "ember serve",
     "test": "ember test",
+    "test:ts": "tsc --noEmit",
     "test:types": "ember try:each --config-path=ember-try-typescript.js",
     "test:all": "ember try:each",
-    "prepublishOnly": "ember ts:precompile",
-    "postpublish": "ember ts:clean"
+    "prepublish:types": "ember ts:precompile",
+    "prepublish:downlevel": "downlevel-dts . ts3.4",
+    "prepublishOnly": "run-s prepublish:types prepublish:downlevel",
```

Comment:
FWIW, I generally prefer the longer `npm-run-all` name (which is serial by default) because:

1) the name is an exact match for a package listed in this very file
2) searching for that name turns up immediate results pointing to the package

Reply:
Ah, so `npm-run-all --serial`? 👍

Reply:
Oh, reading fail—I didn't realize it was serial by default. 👍